test/integration/cms/api/files_test.rb

```diff
@@ -64,6 +64,17 @@ def attachment_for_upload
         )
       end
 
+      test 'index filters by section_id' do
+        section = FactoryBot.create(:cms_section, provider: @provider, parent: @provider.sections.root)
+        5.times { FactoryBot.create(:cms_file, provider: @provider) }
+        2.times { FactoryBot.create(:cms_file, provider: @provider, section_id: section.id) }
```

Comment:
We can use [create_list](https://www.rubydoc.info/gems/factory_bot/FactoryBot%2FSyntax%2FMethods:create_list) for these cases:
```suggestion
        FactoryBot.create(:cms_file, 5, provider: @provider)
        FactoryBot.create(:cms_file, 2, provider: @provider, section_id: section.id)
```

Reply:
Done